### DISCUSSION TITLE
zh/docs/user/multi-factor-authentication

### DISCUSSION THREAD

**Discussion by giscus** [casdoor/casdoor#1821] [Docs comments]
# zh/docs/user/multi-factor-authentication

Config adapter and basic CRUD to policy

https://casdoor.org/zh/docs/user/multi-factor-authentication

**Comment by baiyunlqy** [comment]
Has this feature been removed? I can't find this setting after deployment.

**Comment by hsluoyz** [comment]
@baiyunlqy please use a version >= `v1.309.0`: https://github.com/casdoor/casdoor/pull/1800#issuecomment-1536403376
